src/schematools/contrib/django/management/commands/create_views.py

```diff
@@ -0,0 +1,195 @@
+import re
+from collections import abc, defaultdict
+from typing import Iterable, List, Optional
+
+import requests
+import sqlparse
+from django.core.management import BaseCommand, CommandError
+from django.db import DatabaseError, connection, router, transaction
+
+from schematools.contrib.django.factories import schema_models_factory
+from schematools.contrib.django.models import Dataset, DatasetTableSchema
+from schematools.naming import to_snake_case
+
+DATASETS = Dataset.objects.db_enabled()
+
+
+class Command(BaseCommand):
+    help = "Create the views based on the uploaded Amsterdam schema's."
+    requires_system_checks = []  # don't test URLs (which create models)
+
+    def handle(self, *args, **options):
+        create_views(self, Dataset.objects.db_enabled())
+
+
+def _is_valid_sql(sql: str) -> bool:
+    """Try and execute the SQL return true if it succeeds, rollback the transaction."""
+    try:
+        with transaction.atomic():
+            sid = transaction.savepoint()
+            with connection.cursor() as cursor:
+                cursor.execute(sql)
+            transaction.savepoint_rollback(sid)
+    except:
+        return False
+    return True
+
+
+def _get_scopes(datasetname: str, tablename: str) -> str:
+    all_scopes = []
+    datasetname = to_snake_case(datasetname)
+    tables = DATASETS.get(name=datasetname).schema.tables
+    for table in tables:
```

Comment:
Maybe not for now, but I think this could be implemented somewhat easier with sets.

Reply:
Something along these lines:
```
def _get_scopes2(datasetname: str, tablename: str) -> frozenset[str]:
    from operator import __or__
    from functools import reduce
    dataset = DATASETS.get(name=datasetname).schema
    table = dataset.get_table_by_id(tablename)
    return table.auth | reduce(__or__, [f.auth for f in table.fields])
```